chain/chain/src/chain.rs

```diff
@@ -3524,8 +3524,12 @@ impl Chain {
         }
         let epoch_id = block_header.epoch_id();
         // Use epoch manager because block is not in DB yet.
+        // Let "epoch" be the epoch at which we enable stateless validation.
+        // We need to start storing the state witness from the last block of the previous epoch.
```

Comment:
nit. storing -> generating/sending

Reply:
Let me go with "producing" as that's the wording of this function call.